server/src/main/java/keywhiz/inject/ContextModule.java

```diff
@@ -0,0 +1,23 @@
+package keywhiz.inject;
+
+import com.google.inject.AbstractModule;
+import io.dropwizard.Configuration;
+import io.dropwizard.setup.Environment;
+import keywhiz.KeywhizConfig;
+
+public class ContextModule extends AbstractModule {
+  private final KeywhizConfig config;
+  private final Environment environment;
+
+  public ContextModule(KeywhizConfig config, Environment environment) {
+    this.config = config;
+    this.environment = environment;
+  }
+
+  @Override protected void configure() {
+    // TODO(justin): Consider https://github.com/HubSpot/dropwizard-guice.
+    bind(Environment.class).toInstance(environment);
+    bind(Configuration.class).toInstance(config);
+    bind(KeywhizConfig.class).toInstance(config);
+  }
```

Comment:
Small, single-focus modules are easier to reason about, compose, and test.

Reply:
I kept it just in case. My guess is that whatever that link points to is some sort of first-class Guice-DropWizard integration where we wouldn't have to make these bindings ourselves, and while that does sound like an improvement, I didn't want to pursue it but I also didn't want to lose track of it. Of course, the right thing to do would be to create an internal task and remove the comment, so, maybe I'll just do that.